payment/src/main/resources/application.properties

```diff
@@ -24,7 +25,8 @@ spring.datasource.password=admin
 spring.jpa.properties.hibernate.dialect = org.hibernate.dialect.PostgreSQLDialect
 
 # Hibernate ddl auto (none, create, create-drop, validate, update)
-spring.jpa.hibernate.ddl-auto = none
+spring.jpa.hibernate.ddl-auto=update
```

Comment:
From what I know, we're using Liquibase to manage the database schema, so I'm wondering if setting spring.jpa.hibernate.ddl-auto=update has any effect. I'd appreciate your thoughts on this.

Reply:
Yah, it's unnecessary, when we use `liquibase`. I'll update.